torch/csrc/jit/function_schema.h

```diff
@@ -169,6 +169,9 @@ inline std::ostream& operator<<(std::ostream& out, const FunctionSchema& schema)
       out << "*, ";
       seen_kwarg_only = true;
     }
+    if (schema.is_vararg() && i == schema.arguments().size() - 1) {
```

Comment:
Hm. I think it would be cleaner overall if we gave `Argument` the information that it is vararg. That way we can just stream the argument and the right thing will happen, similar to how it works for default args. 

Then `validate()` can set up the invariant that if there's a vararg it's last in the schema, `schema.is_vararg` can just check whether the last argument is vararg.

Reply:
That makes more sense for this case, but that would need a lot of extra code around all the `prim` operators that have a schema with no arguments + varargs for the schema turned on